modules/core/src/lib/layer-extension.js

```diff
@@ -62,6 +63,21 @@ export class LayerExtension {
 
   initializeState(context, extension) {}
 
+  // Called with old and current props to determine if layer needs to redraw
+  needsRedraw(opts) {
+    const {redrawProps} = this;
```

Comment:
Can we just use the default props of the current extension? What is the use case where we need a separate list?